integration/daemon/daemon_test.go

```diff
@@ -431,16 +432,45 @@ func testLiveRestoreVolumeReferences(t *testing.T) {
 			Source: v.Name,
 			Target: "/foo",
 		}
-		cID := container.Run(ctx, t, c, container.WithMount(m), container.WithCmd("top"))
+
+		cID := container.Run(ctx, t, c, container.WithMount(m), container.WithCmd("sh", "-c", "while true; do echo $RANDOM>>/foo/test.txt; sleep 0.5; done"))
```

Comment:
we're creating a file and continuously add random values to it; I guess this is just to keep the container running, because we don't actually _check_ the content of the file.

Do we need something in the file so that we can verify "it's actually the same file"?

Thinking of a scenario where we _would_ unmount (and re-mount) something; potentially the container would still be happily writing "somewhere", and we don't detect if it's the same mount (and file).

(Perhaps too theoretical, just thinking out loud here 😅)

Reply:
Done.